tests/client/core/api/test_api.js

```diff
@@ -187,4 +188,21 @@ describe('api', () => {
         });
     });
   });
+
+  describe('startLoginUrl', () => {
+    it('includes the next path', () => {
+      const location = { pathname: '/foo', query: { bar: 'BAR' } };
+      assert.equal(
+        api.startLoginUrl({ location }),
+        'https://addons.mozilla.org/api/v3/accounts/login/start/?to=%2Ffoo%3Fbar%3DBAR');
```

Comment:
I don't think the order of query string parameters is predictable when passed through [stringify()](https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options) so this test might fail one day. The annoying way to do it is to use `url.parse()` and deep compare objects instead. However, we can always switch to that on the day it fails, no biggie.